subgraph.xdai.yaml

```diff
@@ -4,7 +4,7 @@ schema:
 dataSources:
   - kind: ethereum/contract
     name: ERC20FeeProxy
-    network: xdai
+    network: gnosis
```

Comment:
This was transparent on the CI because it is done during `yarn install`. We should persist the change in the repository (gnosis is the name of the network in Subgraph Studio).